mtlpy/templates/base.html

```diff
@@ -18,15 +18,18 @@
     </head>
 
     <body class="{% block body_class %}{% endblock %}">
-      <div id="fb-root"></div>
-      <script>(function(d, s, id) {
-        var js, fjs = d.getElementsByTagName(s)[0];
-        if (d.getElementById(id)) return;
-        js = d.createElement(s); js.id = id;
-        js.src = "//connect.facebook.net/{% if LANGUAGE_CODE == 'fr' %}fr_CA{% else %}en_US{% endif %}/all.js#xfbml=1&appId=424645814265696";
-        fjs.parentNode.insertBefore(js, fjs);
-      }(document, 'script', 'facebook-jssdk'));
-      </script>
+
+        <!-- Load Facebook SDK for JavaScript -->
+        <div id="fb-root"></div>
+        <script>(function(d, s, id) {
+            var js, fjs = d.getElementsByTagName(s)[0];
+            if (d.getElementById(id)) return;
+            js = d.createElement(s); js.id = id;
+            js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1";
```

Comment:
The [doc](https://developers.facebook.com/docs/plugins/share-button) doesn't mention anything like that.

We are simply using the sdk for the share/like button AFAIK.